src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.html

```diff
@@ -0,0 +1,92 @@
+<h1>Student Activity Logs</h1>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course. The earliest date you can search for is <b>30 days</b> before today.</p>
+<hr/>
+
+<div *tmIsLoading="isLoading" class="card bg-form no-border">
+  <div class="card-body">
+    <div class="row form-group">
+      <div class="col-4">
+        <div class="col-12">
+          <label for="course-id-dropdown" class="font-weight-bold">Course ID</label>
+        </div>
+        <div class="col-12">
+          <select id="course-id-dropdown" class="form-control col-8" [ngModel]="formModel.courseId" (ngModelChange)="updateSearchButtonDisabled($event)">
```

Comment:
Some kind of obvious indication that this field is compulsary?  Maybe something like `---Select Course ID---`